server/licences/licenceStatus.ts

```diff
@@ -45,7 +45,7 @@ const statusConfig: Record<LicenceStatus, LicenceStatusConfig> = {
   NOT_STARTED: {
     label: 'Not started',
     description: 'Not started',
-    colour: 'grey',
+    colour: 'yellow',
```

Comment:
is this used across prisons and probation? 
This presumably would be active even if the feature was disabled - so released immediately?

Should we hide this change behind the toggle too?